dlpy/tests/datasources/sample_syntax_for_test.json

```diff
@@ -18,8 +18,8 @@
   "inputlayer7":{"name": "data", "layer": {"offsets": [0, 0, 0], "height": 150, "width": 100, "nchannels": 3, "type": "input"}},
   "convo1":{"name": "convo1", "layer": {"act": "relu", "stride":1, "height": 3, "width": 3, "nfilters": 10, "includebias":true, "type": "convo"}, "srclayers": ["input1"]},
   "convo2":{"name": "convo2", "layer": {"act": "relu", "height": 7, "stride": 1, "width": 5, "nfilters": 32, "includebias":true, "type": "convo"}, "srclayers": ["input1"]},
-  "pool1": {"name": "pool1", "layer": {"pool": "max", "stride": 2, "height": 2, "width": 2, "type": "pool"}, "srclayers": ["conv"]},
-  "pool2": {"name": "pool2", "layer": {"pool": "max", "stride": 3, "height": 2, "width": 3, "type": "pool"}, "srclayers": ["conv"]},
+  "pool1": {"name": "pool1", "layer": {"pool": "max", "stridevertical": 2, "stridehorizontal": 2, "height": 2, "width": 2, "type": "pool"}, "srclayers": ["conv"]},
+  "pool2": {"name": "pool2", "layer": {"pool": "max", "stridevertical": 2, "stridehorizontal": 3, "height": 2, "width": 3, "type": "pool"}, "srclayers": ["conv"]},
```

Comment:
so verticals are 2
but horizontals are 2 and 3.
check out again to see if there is an error here

Reply:
dlpy/tests/datasources/sample_syntax_for_test.json: change pool1 and pool2 results, as stride is defaulted to pooling window size which is consistent with Keras API.
pool1 stridevertical and stridehorizeontal are 2 since default stride is 2. 
pool2 strides are different since pooling windows have width=3 and height=2.
Here are test cases:
In pool1:
dict1 = Pooling(name='pool1', src_layers=[ Conv2d(n_filters=3, name='conv')]).to_model_params()
In pool2:
dict1 = Pooling(name='pool2', width=3, height=2,
                        src_layers=[Conv2d(n_filters=3, name='conv')]).to_model_params()